docs/integrations/sources/shopify-migrations.md

```diff
@@ -1,5 +1,17 @@
 # Shopify Migration Guide
 
+## Upgrading to 2.1.0
+This version implements `Shopify GraphQL BULK Operations` to speed up the following streams:
```

Comment:
This is not a breaking change because we `added` a new property: `presentment_prices` that contains the `compare_at_prices` as the `object` with `amount` and `currency_code` nested properties. 

The `presentment_prices` was never in the `REST` output, but the schema had an invalid reference though, I've corrected it up to the docs.

The original `compare_at_prices` didn't change (at the root of the schema); it's still a `string.`

Reply:
I'm confused: [this line](https://github.com/airbytehq/airbyte/pull/37767/files#diff-93f120238b8f5d9fda0d64fcfb955b38f71fae919523c262e5f4731a332ab3d9R124) seems to indicate to me that it was previously a number and is now an object. What am I missing?

Reply:
If you look at the old `expected_records,`, you will not find the `presentment_prices` included as additional proof.

Reply:
I'm not sure I follow. Can you give me an example of value for `compare_at_prices` before the change? Our expected records do not populate this value

Reply:
I'll add the description about it to the migration guide.